torch/csrc/jit/passes/quantization.cpp

```diff
@@ -194,6 +199,21 @@ bool nodeQuantizable(Node* n) {
     });
 }
 
+bool userDefinedCallFunction(Node* n) {
```

Comment:
Why is this called user defined? Isnt this a function that is not one of the functions listed in the arguments of the call? This can be a aten function isnt it?

Reply:
there are "builtin" CallFunction that we don't want to go through their graph and do quantization because we want to preserve the boundary, e.g. `linear`